### ISSUE TITLE
Testcase getUserBlogs is failing.

### ISSUE THREAD

**Issue by moelrobii** [AminoJS/Amino.JS#44]
**Describe the bug**
We noticed that a Testcase is failing in our testing for Release 3.0. I think this is due to a backend change in Narvii's Systems. 

**To Reproduce**
Login a client with the AminoJS Libary, then call the Function:
`<AminoJS>.getUserBlogs(comID, userID, count);`
**OR**
start the Testcases with `npm test`

The Bug should then be visible there.

**Expected behavior**
The `<calledVariable>.blogs` should be an array. The testing system says that it's not.

**Screenshots**
Here is the testcase result with another account:

![code_2018-10-22_16-38-56](https://user-images.githubusercontent.com/44272882/47298802-facb8080-d618-11e8-9ac6-a014246f433a.png)

And here is the testcase with the testing Account and more debug logs. (JSON in text under screenshot)

![code_2018-10-22_16-42-43](https://user-images.githubusercontent.com/44272882/47298964-60b80800-d619-11e8-91e4-b775599b7bdb.png)

```json
== Raw JSON response==
{
  "blogs": null,
  "status": "ok",
  "error": null
}
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Node-Version: v.8.12.0
 - Version: latest master

**Comment by akatsukilevi** [comment]
I will be working to fix it

**Comment by felixfong227** [comment]
Funny thing is when I try to run the test locally, everything works without anything sort of error
![image](https://user-images.githubusercontent.com/13918481/47511163-33cf5380-d8ac-11e8-9be8-93cd766efda4.png)
But when I kick start an other build with the same branch and same commit
![image](https://user-images.githubusercontent.com/13918481/47511238-5eb9a780-d8ac-11e8-9c4d-3c9362f11d58.png)
https://travis-ci.org/AminoJS/Amino.JS/builds/439469084
🤔

**Comment by moelrobii** [comment]
Could it be that the User we are loading does not have any Userblogs? Which is impossible to be Honest da Travis itself a Blog creates. But you know it’s a thought.

**Comment by felixfong227** [comment]
@moelrobii 
> Could it be that the User we are loading does not have any Userblobs

Haven’t really taken that as consideration 🤣

Well, should we just make a default return value as `null` and inside the test case, accept both array and null?

**Comment by moelrobii** [comment]
There is a thing already in Place that does this. :) 

Check modules/getUserBlogs.js

**Comment by felixfong227** [comment]
Ya, thanks @moelrobii 
https://github.com/AminoJS/Amino.JS/blob/37e5e544b9151f223b951642ab632e2d600ef9ba/modules/getUserBlogs.js#L29
😂 
So now we just need to make the test case to also accept `null` as well 👍

**Comment by moelrobii** [comment]
Closed. Fixed with PR #41 and commit Hash bf34a278f0751bd66d0a2beae70df38d21fe186b
